frontend/src/app/main/ui/viewer/header.cljs

```diff
@@ -158,88 +306,176 @@
-        (for [id (get-in file [:data :pages])]
-          [:li {:id (str id)
-                :key (str id)
-                :on-click (partial navigate-to id)}
-           (get-in file [:data :pages-index id :name])])]]]
-
-     [:span.icon {:on-click open-dropdown} i/arrow-down]
-     [:div.current-frame
-      {:on-click toggle-thumbnails}
-      [:span.label "/"]
-      [:span.label frame-name]]
-     [:span.icon {:on-click toggle-thumbnails} i/arrow-down]]))
-
+    (if new-css-system
+      [:div {:class (stl/css :sitemap-zone)
+             :title (tr "viewer.header.sitemap")}
+       [:span {:class (stl/css :project-name)} project-name]
+       [:div {:class (stl/css :sitemap-text)}
+        [:div {:class (stl/css :breadcrumb)
+               :on-click open-dropdown}
+         [:span  {:class (stl/css :breadcrumb-text)}
+          (dm/str file-name " / " page-name)]
+         [:span {:class (stl/css :icon)} i/arrow-refactor]
+         [:span "/"]
+         [:& dropdown {:show @show-dropdown?
+                       :on-close close-dropdown}
+          [:ul {:class (stl/css :dropdown-sitemap)}
+           (for [id (get-in file [:data :pages])]
+             [:li {:class (stl/css-case :dropdown-element true
+                                        :selected (= page-id id))
+                   :id (str id)
+                   :key (str id)
+                   :on-click (partial navigate-to id)}
+              [:span {:class (stl/css :label)}
+               (get-in file [:data :pages-index id :name])]
+              (when (= page-id id)
+                [:span {:class (stl/css :icon-check)} i/tick-refactor])])]]]
+        [:div {:class (stl/css :current-frame)
+               :on-click toggle-thumbnails}
+         [:span {:class (stl/css :frame-name)} frame-name]
+         [:span {:class (stl/css :icon)} i/arrow-refactor]]]]
+
+
+      ;; OLD
+      [:div.sitemap-zone {:alt (tr "viewer.header.sitemap")}
+       [:div.breadcrumb
+        {:on-click open-dropdown}
+        [:span.project-name project-name]
+        [:span "/"]
+        [:span.file-name file-name]
+        [:span "/"]
+
+        [:span.page-name page-name]
+        [:& dropdown {:show @show-dropdown?
+                      :on-close close-dropdown}
+         [:ul.dropdown
+          (for [id (get-in file [:data :pages])]
+            [:li {:id (str id)
+                  :key (str id)
+                  :on-click (partial navigate-to id)}
+             (get-in file [:data :pages-index id :name])])]]]
+
+       [:span.icon {:on-click open-dropdown} i/arrow-down]
+       [:div.current-frame
+        {:on-click toggle-thumbnails}
+        [:span.label "/"]
+        [:span.label frame-name]]
+       [:span.icon {:on-click toggle-thumbnails} i/arrow-down]])))
 
 (mf/defc header
   [{:keys [project file page frame zoom section permissions index interactions-mode]}]
-  (let [go-to-dashboard
+  (let [new-css-system (mf/use-ctx ctx/new-css-system)
+        go-to-dashboard
```

Comment:
needs `mf/use-fn`

Reply:
go-to-dashboard
        (mf/use-fn
         #(st/emit! (dv/go-to-dashboard)))

Reply:
Added